tests/unit/test_annotation.py

```diff
@@ -29,3 +29,25 @@ def test_get_points(self, fxt_ellipses: List[Ellipse]):
             analytical_area = ellipse.get_area()
             numerical_area = sg.Polygon(ellipse.get_points(num_points=360 * 10)).area
             assert np.abs(analytical_area - numerical_area) < 1e-6
+
+
+class HashKeyTest:
+    @pytest.fixture
+    def fxt_hashkeys_same(self):
+        hash_key = np.random.randint(0, 256, size=(1, 64), dtype=np.uint8)
+        hashkey1 = HashKey(hash_key=hash_key)
+        hashkey2 = HashKey(hash_key=hash_key)
+        return hashkey1, hashkey2
+
+    @pytest.fixture
+    def fxt_hashkeys_diff(self):
+        hashkey1 = HashKey(hash_key=np.random.randint(0, 256, size=(1, 64), dtype=np.uint8))
+        hashkey2 = HashKey(hash_key=np.random.randint(0, 256, size=(1, 64), dtype=np.uint8))
+        return hashkey1, hashkey2
```

Comment:
```suggestion
    def fxt_hashkeys_diff(self):
        np.random.seed(3003)
        hashkey1 = HashKey(hash_key=np.random.randint(0, 256, size=(1, 64), dtype=np.uint8))
        hashkey2 = HashKey(hash_key=np.random.randint(0, 256, size=(1, 64), dtype=np.uint8))
        return hashkey1, hashkey2
```
There was a possibility that both are equal by change.

Reply:
Fix in [f34ca30](https://github.com/openvinotoolkit/datumaro/commit/f34ca30dd441a0455ebc3fb7e2c0175bcfca5680)